### DISCUSSION TITLE
use toJS()  can not get  JavaScript  Object

### DISCUSSION THREAD

**Discussion by yumo-mt** [mobxjs/mobx#3701] [Q&A]
mobx version: 6.3.1

```

  configure({
    enforceActions: 'always',
  })
  
  class Store {
    constructor() {
      makeAutoObservable(this)
    }
    model = observable.map({
      name: 'Alex',
    })
  }
  
  const store = new Store()
  
  const Demo = () => {
    console.log(toJS(store.model))  //  Output:  Map(1) {'name' => 'Alex'}
  }
  
  Demo()

```

i want to use toJS api to get Object   { name: 'Alex' }， but  not  work.

mobx@3.x can do this。

Has the use of "toJS" changed ？

**Comment by urugator** [comment]
> Has the use of "toJS" changed ？

Yes. `ObservableMap`s are converted to non-observable native `Map`s:
https://github.com/mobxjs/mobx/blob/7cdc7ecd6947a6da10f10d2e4a1305297b816007/packages/mobx/src/api/tojs.ts#L48-L53
